### DISCUSSION TITLE
How to disable giscus comments on a page?

### DISCUSSION THREAD

**Discussion by ghost** [HEIGE-PCloud/DoIt#288] [Q&A] [answered]
Hey there.
I have searched high and low but I cannot find how to disable comments on a page. 
I have read some posts suggesting adding commentable: false to the front matter but this does not work.

Thanks for your help!

**Comment by HEIGE-PCloud** [comment, marked as answer]
Add

```yaml
comment:
  enable: false
```

to your front matter can disable the comment on this page.

Reference: https://hugodoit.pages.dev/theme-documentation-content/#front-matter

**Reply by ghost** [reply]
Face palm moment when I saw your response! Thank you for help and the awesome theme!
